tensorflow/python/kernel_tests/array_ops_test.py

```diff
@@ -507,6 +507,19 @@ def __init__(self, test, x, tensor_type=dtypes.int32, check_type_infer=True):
 
   def __getitem__(self, spec):
     op = self.x.__getitem__(spec)
+
+    def eval_if_tensor(x):
```

Comment:
I believe this is TestCase.evaluate in TF

Reply:
@alextp, I do not understand your comment. Here I simply move this simple eval_if_tensor() function from the original position line 518 to line 510, such that the following codes line 517-521(corresponds to the test for boolean_mask x[x>a]) can call the eval_if_tensor() function.